src/postgraphile/http/__tests__/createPostGraphileHttpRequestHandler-test.js

```diff
@@ -1,855 +0,0 @@
-          'foo.number': 42,
-        },
-      })
-      await request(server)
-        .post('/graphql')
-        .send({ query: '{hello}' })
-        .expect(200)
-        .expect('Content-Type', /json/)
-        .expect({ data: { hello: 'world' } })
-      expect(pgPool.connect.mock.calls).toEqual([[]])
-      expect(pgClient.query.mock.calls).toEqual([
-        ['begin'],
-        [
-          {
-            text: 'select set_config($1, $2, true), set_config($3, $4, true)',
-            values: ['foo.string', 'test1', 'foo.number', '42'],
-          },
-        ],
-        ['commit'],
-      ])
-      expect(pgClient.release.mock.calls).toEqual([[]])
-    })
-
-    test('will correctly hand over pgSettings function to the withPostGraphileContext call', async () => {
-      pgPool.connect.mockClear()
-      pgClient.query.mockClear()
-      pgClient.release.mockClear()
-      const server = createServer({
-        pgSettings: req => ({
-          'foo.string': 'test1',
-          'foo.number': 42,
-        }),
-      })
-      await request(server)
-        .post('/graphql')
-        .send({ query: '{hello}' })
-        .expect(200)
-        .expect('Content-Type', /json/)
-        .expect({ data: { hello: 'world' } })
-      expect(pgPool.connect.mock.calls).toEqual([[]])
-      expect(pgClient.query.mock.calls).toEqual([
-        ['begin'],
-        [
-          {
-            text: 'select set_config($1, $2, true), set_config($3, $4, true)',
-            values: ['foo.string', 'test1', 'foo.number', '42'],
-          },
-        ],
-        ['commit'],
-      ])
-      expect(pgClient.release.mock.calls).toEqual([[]])
-    })
-
-    test('will call additionalGraphQLContextFromRequest if provided and add the response to the context', async () => {
-      const helloResolver = jest.fn(
-        (source, args, context) => context.additional,
-      )
-      const contextCheckGqlSchema = new GraphQLSchema({
-        query: new GraphQLObjectType({
-          name: 'Query',
-          fields: {
-            hello: {
-              type: GraphQLString,
-              resolve: helloResolver,
-            },
-          },
-        }),
-      })
-      const additionalGraphQLContextFromRequest = jest.fn(() => ({
-        additional: 'foo',
-      }))
-      const server = createServer({
-        additionalGraphQLContextFromRequest,
-        getGqlSchema: () => Promise.resolve(contextCheckGqlSchema),
-      })
-
-      await request(server)
-        .post('/graphql')
-        .send({ query: '{hello}' })
-        .expect(200)
-        .expect('Content-Type', /json/)
-        .expect({ data: { hello: 'foo' } })
-      expect(additionalGraphQLContextFromRequest).toHaveBeenCalledTimes(1)
-      expect(
-        additionalGraphQLContextFromRequest.mock.calls[0][0],
-      ).toBeInstanceOf(http.IncomingMessage)
-      expect(
-        additionalGraphQLContextFromRequest.mock.calls[0][1],
-      ).toBeInstanceOf(http.ServerResponse)
-    })
-  })
-}
```

Comment:
I don't think you've replaced this test file after deleting it?

Reply:
(This is possibly where 855 of the removed 1K LOC came from 😳 )

Reply:
oh brother.  🤦‍♂️ 

0ab968a